frontend/src/app/hello/detail/detail.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import {
+  HelloServiceClient,
+  Hello,
+  HelloById,
+} from 'src/app/proto/hello/src/hello/hello.pb';
+
+@Component({
+  selector: 'app-detail',
+  templateUrl: './detail.component.html',
+  styleUrls: ['./detail.component.scss'],
+})
+export class DetailComponent implements OnInit {
+  private detailId: number;
+  hello$: Hello;
+
+  constructor(
+    private route: ActivatedRoute,
+    private helloService: HelloServiceClient,
+  ) {}
+
+  ngOnInit(): void {
+    this.route.paramMap.subscribe(params => {
+      this.detailId = Number(params.get('id'));
+    });
+
+    this.helloService
+      .findOne(new HelloById({ id: this.detailId }))
+      .toPromise()
```

Comment:
Please use RxJS observables with `.subscribe()` here. You can also use the Angular async pipe to handle unsubscribing automatically (it's actually the preferred way).

Reply:
I don't know how to do it with `.pipe()` it just dont show results. With `subscribe` it is working.